src/builders_v2/linux_vm/nvidia.rs

```diff
@@ -36,53 +30,65 @@ pub enum NvidiaError {
     Io(#[from] std::io::Error), // Fallback error for generic I/O errors
 }
 
+// hardcoded for now - we always build this version of driver
+/// Version of the installed driver (constant because we only use this version now).
+const DRIVER_VERSION: &str = "550.120";
```

Comment:
Let's have a reminder somewhere to revisit this once the 560.x series gets stable (it's in beta right now).

Reply:
I created a ticket in our Linear for that